### DISCUSSION TITLE
Symfony Cannot Find Docker Compose

### DISCUSSION THREAD

**Discussion by vonCorax** [symfony/symfony#46223] [Symfony Book: The Fast Track]
I'm starting into the Book, and I've already hit a snag. I've installed docker and docker compose - `docker` reports version 20.10.14 and `compose` reports version 2.4.1, and `docker run hello-world` works - but `symfony book:check-requirements` reports "Cannot find Docker Compose, please install it." What have I missed? I'm running PHP 8.1 on Ubuntu 18.04.

**Comment by DylanKas** [comment]
You need to install docker-composer 
https://docs.docker.com/compose/install/

**Comment by ArnaudLier** [comment]
You can use `docker compose` instead of `docker-compose` :smiley:
